### PR TITLE
AT_05.07 _002| Verify New Item page contains input field

### PR DESCRIPTION
**Pull request by Tatsiana-Nazarava**
https://trello.com/c/tEhtZAd2/381-at0507-002-verify-new-item-button-is-clickable-on-the-main-page

### CODE REVIEW

File: cypress/e2e/newItemInputField.cy.js
Comment (Paul-Korz) [inline cypress/e2e/newItemInputField.cy.js:1]: Please rename describe to 'New item | Input field'. 
I had the same comment.

Comment (Paul-Korz) [reply cypress/e2e/newItemInputField.cy.js:1]: If I understand correctly, these 'IT' are same. Maybe you should left only one of them?

Comment (Tatsiana-Nazarava) [reply cypress/e2e/newItemInputField.cy.js:1]: @Paul-Korz  the first 'it' isn't mine. This functionality has already been tested by another member. I also check it. And I add my test to the already created file for the same user story.

**Review by Paul-Korz — APPROVED** [review]

File: cypress/e2e/newItemInputField.cy.js [hunk 1]
```diff
@@ -7,4 +7,10 @@ describe('New Item', () => {
         cy.url().should('include', '/view/all/newJob')
         cy.get('.jenkins-input').should('be.visible')
     })
+    
+    it('AT_05.07 _002 | Verify New Item page contains input field', () => {
+        cy.get('a[href ="/view/all/newJob"]').click()
+        cy.url().should('eq', 'http://localhost:8080/view/all/newJob')
```
Comment (Rom-T) [inline cypress/e2e/newItemInputField.cy.js:13]: The port is better not to hardcode. Because it may differ. As an example, see in #71 or #86

**Review by Rom-T — APPROVED** [review]

**Review by SawkaQA — APPROVED** [review]

**Pull request merged**